frontend/lib/pages/entry_page.dart

```diff
@@ -28,6 +29,31 @@ class _EntriesPageState extends State<EntryPage> {
   }
 
   var client = HttpClientService();
+
+  Future showDialog2() {
```

Comment:
I don't think showDialog2 is the best name for a function

Reply:
Could you also add a screenshot of the page?

Reply:
yea totally right!

Reply:
<img width="337" alt="Screen Shot 2020-09-15 at 10 08 30 AM" src="https://user-images.githubusercontent.com/46607090/93221276-93060d80-f73b-11ea-829f-c88152f3e67f.png">
<img width="334" alt="Screen Shot 2020-09-15 at 10 08 24 AM" src="https://user-images.githubusercontent.com/46607090/93221283-95686780-f73b-11ea-927e-80743901d9cd.png">

Reply:
agreed, dialog2 could be better named